source/_usecases/find the Index of an element in an Array

```diff
@@ -0,0 +1,17 @@
+extends: _layouts.usecase
+date: 2018-03-27
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex
+reference: find the Index of an element in an Array
```

Comment:
Please change to `reference: Array.indexOf` which is the name of the method we're using